src/components/UI/atoms/Button/Button.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import widths from '<variables>/width';
+import themeBorderRadius from '<variables>/border';
+import heights from '<variables>/height';
+import { buttonColors } from '<variables>/colorPalette';
+import { fontSizes } from '<variables>/fonts';
+
+/**
+ * @description - Button Component
+ *
+ * @prop {string} display - display
+ * @prop {string} hoverBackground - hoverBackground
+ * @prop {string} hoverColor - hoverColor
+ * @prop {string} fontSize - font Size
+ * @prop {string} borderColor - border
+ * @prop {string} borderRadius - border radius
+ * @prop {string} width - width
+ * @prop {string} height - height
+ * @prop {string} color - color
+ * @prop {function} onClick - onClick
+ * @prop {string} backgroundColor - background color
+ * @prop {children} children - children
+ *
+ * @return {component} Button
+ */
+const Button = ({
+  hoverBackground,
+  hoverColor,
+  onClick,
+  borderColor,
+  children,
+  display,
+  color,
+  fontSize,
+  width,
+  height,
+  backgroundColor,
+  borderRadius,
+}) => (
+  <Button.Container
+    hoverColor={hoverColor}
+    hoverBackground={hoverBackground}
+    width={width}
+    height={height}
+    borderColor={borderColor}
```

Comment:
Are the borderColor going to change elsewhere in the designs

Reply:
Yes, the get started button has a different borderColor, from the other buttons, because of the color scheme used for that button.
Thank you.

Reply:
Use the buttonType to render the borderColor